cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -207,3 +208,7 @@
     display: none;
   } );
 }
+
+#ctrl-f-modal h3 {
```

Comment:
Does `ctrl-f-modal` ID come in through the `ctrl-f` package? All I'm seeing set in the markup for an ID in this PR is `id="ctrl-f"`.

Reply:
Yeah it's in the [ctrl-f package](https://github.com/cfpb/ctrl-f/blob/1d17eb0e77c8ebc63c94b769adf666083a2d2878/src/Modal.tsx#L83) and the sizing of the h3 is the only thing cf.gov overrides.